gulpfile.js

```diff
@@ -40,7 +40,7 @@ var filenameLowercase = 'mist';
 var filenameUppercase = 'Mist';
 var applicationName = 'Mist'; 
 var electronVersion = '1.2.5';
-var gethVersion = '1.4.10';
+module.exports.gethVersion = '1.4.10';
```

Comment:
My fix for the Linux node download issue will replace this with something better.